packages/jest-transform/src/index.ts

```diff
@@ -1,22 +1,42 @@
 import type { Transformer } from '@jest/transform';
-import { transformSync, getPackageInfo } from '@vanilla-extract/integration';
+import { setAdapter } from '@vanilla-extract/css/adapter';
+import {
+  transformSync,
+  getPackageInfo,
+  IdentifierOption,
+} from '@vanilla-extract/integration';
 import * as esbuild from 'esbuild';
 
-const vanillaTransformer: Transformer = {
+interface TransformerConfig {
+  identOption: IdentifierOption;
+}
+
+const vanillaTransformer: Transformer<TransformerConfig> = {
   canInstrument: false,
   process(source, filePath, options) {
-    const { name: packageName } = getPackageInfo(options.config.rootDir);
+    const { config, supportsStaticESM, transformerConfig } = options;
+    const { identOption = 'debug' } = transformerConfig;
+    const { name: packageName } = getPackageInfo(config.rootDir);
+
+    setAdapter({
```

Comment:
This is part of the problem why I did not get it completely working. As it stands, in my Jest setup, this jest-transform setup loads the main-cjs file of the css/adapter submodule, but the actual `@vanilla-extract/css` imports load the browser bundle. This basically means that this transformer sets an adapter in one file, while the `generateIdentifier` function tries to read the value from another file. Not sure if there is a better way to do what I'd like, but it seemed to be necessary to set an adapter that could be read from this config.